pandas/core/generic.py

```diff
@@ -4949,10 +4949,14 @@ def pipe(self, func, *args, **kwargs):
     Returns
     -------
     DataFrame, Series or scalar
-        If DataFrame.agg is called with a single function, returns a Series
-        If DataFrame.agg is called with several functions, returns a DataFrame
-        If Series.agg is called with single function, returns a scalar
-        If Series.agg is called with several functions, returns a Series.
+
+        The return can be:
+
+        - If DataFrame.agg is called with a single function, returns a Series.
+        - If DataFrame.agg is called with several functions, returns
+          a DataFrame.
+        - If Series.agg is called with single function, returns a scalar.
+        - If Series.agg is called with several functions, returns a Series.
```

Comment:
Let's do the same thing you are doing below, where instead of putting periods at the end of each bullet you end with a short sentence after the bullets